app/src/main/AndroidManifest.xml

```diff
@@ -40,15 +40,15 @@
             </intent-filter>
         </activity>
         <activity
-            android:name=".TUC_Info"
+            android:name=".ui.InfoActivity"
             android:label="TUCInfo">
         </activity>
         <activity
-            android:name=".TUC_Photos"
+            android:name=".ui.PhotosActivity"
             android:label="Photos">
         </activity>
         <activity
-            android:name=".CECH_Info"
+            android:name=".ui.CECHInfoActivity"
```

Comment:
I really recommend organizing your code into packages, and avoiding top-level files as much as possible. According to your UML diagram, these files should be in a `ui` package.

Reply:
Yes, good call.  I'd also make the package names lower case, as that is convention.